sympy/functions/elementary/trigonometric.py

```diff
@@ -1568,6 +1568,97 @@ def taylor_term(n, x, *previous_terms):
                     bernoulli(2*k)*x**(2*k - 1)/factorial(2*k))
 
 
+class sinc(TrigonometricFunction):
+    r"""Represents unnormalized sinc function
+
+    Examples
+    ========
+
+    >>> from sympy import sinc, oo, jn, Product, Symbol
+    >>> from sympy.abc import x
+    >>> sinc(x)
+    sinc(x)
+
+    Automated Evaluation
```

Comment:
Automated Evaluation  --->  * Automated Evaluation 
(It would look better in docs)